### PR TITLE
Fix bip api connectivity issues

### PR DESCRIPTION
**Pull request by tejans24**
## What was the problem?
Resolves cert issues and BIP connectivity issues we are seeing in dev.

Associated tickets or Slack threads:
- #2276 
- #2293
- #2311

## How does this fix it?[^1]
- Updates in the svc-bip-api-certgen.sh script, involving the generation of certificates and key management, requiring `VA-Internal-S2-ICA11`, `VA-Internal-S2-RCA2` certs.
- Changes in `BipApiConfig.java` and `BipApiService.java` for better API service configuration related to ssl configuration  and JWT token generation.
- Modifications in various application.yaml files across different environments (dev, qa, prod-test, prod), mainly related to the claim base URL.
- Added claim issuer attribute to generate JWT tokens

I used this javascript code to verify connectivity as initially

```
const axios = require('axios');
const https = require('https');
const fs = require('fs');

// Path to your .p12 file and its passphrase
const p12Path = "./keystore.p12";
const caPath = "./all_va_internal_cas.pem";

const p12Passphrase = 'password';

// Read the .p12 file
const p12Content = fs.readFileSync(p12Path);
const caCert = fs.readFileSync(caPath); // CA certificate

const agent = new https.Agent({
    pfx: p12Content,
    passphrase: p12Passphrase,
    ca: caCert,
});


// Create an Axios instance
const axiosInstance = axios.create({
    httpsAgent: agent,
});

async function getJwtToken() {
    const tokenUrl = 'https://claims-test.dev.bip.va.gov/api/v1/token';
    const tokenData = {
        userID: "",
        applicationID: "",
        stationID: ""
    };

    try {
        const response = await axiosInstance.post(tokenUrl, tokenData);
        const jwtToken = response.data; 
        console.log(jwtToken)
        // Update Axios instance with the JWT token
        axiosInstance.defaults.headers.common['Authorization'] = `Bearer ${jwtToken}`;
        return jwtToken;
    } catch (error) {
        console.error('Error fetching JWT token:', error);
        return null;
    }
}


async function makeClaimsCall() {
    const claimsUrl = 'https://claims-test.dev.bip.va.gov/api/v1/claims/9666958'; 

    try {
        const response = await axiosInstance.get(claimsUrl);
        console.log(response.data);
    } catch (error) {
        console.error(error);
    }
}


async function main() {
    const jwtToken = await getJwtToken();
    if (jwtToken) {
        await makeClaimsCall();
    } else {
        console.error('Failed to retrieve JWT token');
    }
}

main();
```

### CODE REVIEW

File: svc-bip-api/src/main/java/gov/va/vro/bip/config/BipApiConfig.java [hunk 1]
```diff
@@ -22,13 +21,12 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.security.KeyManagementException;
-import java.security.KeyStore;
-import java.security.KeyStoreException;
-import java.security.NoSuchAlgorithmException;
+import java.security.*;
```
Comment (agile-josiah) [inline svc-bip-api/src/main/java/gov/va/vro/bip/config/BipApiConfig.java:24]: Do we want to use wildcard imports?

**Review by agile-josiah — CHANGES_REQUESTED** [review]
I think everything is fine, I left 1 comment but otherwise LGTM

File: svc-bip-api/src/main/java/gov/va/vro/bip/config/BipApiConfig.java (hunk 1 above)
Comment (tejans24) [reply svc-bip-api/src/main/java/gov/va/vro/bip/config/BipApiConfig.java]: yea not sure why it was a wildcard, IDE might have done it auto.

**Review by msnwatson — APPROVED** [review]

**Review by agile-josiah — APPROVED** [review]
LGTM

**Pull request merged**